core/src/main/java/Scenes/MainMenu.java

```diff
@@ -114,12 +100,14 @@ public boolean touchDown(InputEvent event, float x, float y, int pointer, int bu
     leaderboardsButton.addListener(new InputListener() {
       @Override
       public boolean touchDown(InputEvent event, float x, float y, int pointer, int button) {
+        // Change to leaderboards screen.
         mainGame.setScreen(FilePlayMain.ScreenType.LEADERBOARDS);
         return true;
       }
     });
     settingsButton.addListener(new InputListener() {
       public boolean touchDown(InputEvent event, float x, float y, int pointer, int button) {
+        // Change to the settings screen.
```

Comment:
These new comments are not needed, as it is already clear from the code that the screen is getting changed.  I.e., the reader can see that mainGame's screen is getting set to a SETTINGS screen.